Makefile

```
@@ -0,0 +1,135 @@
+# -- General
+SHELL := /bin/bash
+
+# -- Docker
+# Get the current user ID to use for docker run and docker exec commands
+DOCKER_UID           = $(shell id -u)
+DOCKER_GID           = $(shell id -g)
+DOCKER_USER          = $(DOCKER_UID):$(DOCKER_GID)
+COMPOSE              = DOCKER_USER=$(DOCKER_USER) docker compose
+COMPOSE_RUN          = $(COMPOSE) run --rm
+COMPOSE_RUN_BACK     = $(COMPOSE_RUN) back
+
+# -- Elasticsearch
+ES_PROTOCOL = http
+ES_HOST     = localhost
+ES_PORT     = 9200
+ES_INDEX    = statements
+ES_URL      = $(ES_PROTOCOL)://$(ES_HOST):$(ES_PORT)
+
+# -- WARREN
+WARREN_BACKEND_IMAGE_NAME         ?= warren-back
+WARREN_BACKEND_IMAGE_TAG          ?= development
+WARREN_BACKEND_IMAGE_BUILD_TARGET ?= development
+WARREN_BACKEND_SERVER_PORT        ?= 8100
+
+
+# ==============================================================================
+# RULES
+
+default: help
+
+.env:
+	cp .env.dist .env
+
+
+# -- Docker/compose
+bootstrap: ## bootstrap the project for development
+bootstrap: \
+  .env \
+  build \
+  dev
+.PHONY: bootstrap
+
+build: ## build the app container
+build: .env
+	WARREN_BACKEND_IMAGE_BUILD_TARGET=$(WARREN_BACKEND_IMAGE_BUILD_TARGET) \
+	WARREN_BACKEND_IMAGE_NAME=$(WARREN_BACKEND_IMAGE_NAME) \
+	WARREN_BACKEND_IMAGE_TAG=$(WARREN_BACKEND_IMAGE_TAG) \
+	  $(COMPOSE) build back
+.PHONY: build
+
+down: ## stop and remove backend containers
+	@$(COMPOSE) down
+.PHONY: down
+
+logs-back: ## display backend logs (follow mode)
+	@$(COMPOSE) logs -f back
+.PHONY: logs-back
+
+run: ## run the whole stack
+run: run-back
+.PHONY: run
+
+run-back: ## run LRS server with the runserver backend (development mode)
```

```suggestion
run-back: ## run the backend server (development mode)
```
Is it LRS server or backend server?